modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1757,6 +1757,15 @@ void NativeReloadNode::_notification(int p_what) {
 				}
 
 				for (Map<String, Set<NativeScript *>>::Element *U = NSL->library_script_users.front(); U; U = U->next()) {
+					// backported from godotengine/godot#46514
+					// multiple gdnative libraries may be reloaded
+					// the gdnative library and script path should match
+					// to prevent failing NSL->library_classes lookup from get_script_desc()
+					// in script->_update_placeholder below
```

Comment:
Would be good to improve casing and the lines are wrapped a bit early.
```suggestion
					// Multiple GDNative libraries may be reloaded. The library and script
					// path should match to prevent failing `NSL->library_classes` lookup
					// from `get_script_desc()` in `script->_update_placeholder` below.
```
Though comments that refer to code like this inevitably end up outdated when code gets refactored, it's usually better to paraphrase what the code actually does in plain English. But not critical for this to be merged if the fix is accepted.

Reply:
Makes sense! I just carbon-copied the original comment for consistency.